### PR TITLE
Wire up fetches

### PR DESCRIPTION
**Pull request by mihiramin89**
This PR wires up the fetches. 
When the left drawer is opened, it fetches the service categories from the following endpoint

- `https://mofb-api.appspot.com/api/v1/taxonomy/10/children`

When a category is opened, it fetches the sub categories from the following endpoint

- `https://mofb-api.appspot.com/api/v1/taxonomy/11/children`

When a sub category is checked, it fetches the agency information from the following endpoint

- `https://mofb-api.appspot.com/api/v1/agency?taxonomyId=11`

### **Screenshots of the new UI**

cursor_and_service_locator

cursor_and_service_locator

cursor_and_service_locator_and_service_locator_-__private_browsing_

### **Will have follow up PR for cleaning up the UI & adding unit tests**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -134,6 +122,7 @@ module.exports = ({ prod } = {}) => {
       new webpack.DefinePlugin({
         'process.env': {
           NODE_ENV: JSON.stringify(process.env.NODE_ENV),
+          MAP_KEY: JSON.stringify(process.env.MAP_KEY),
```
Comment (mihiramin89) [inline webpack.config.js]: I think this is the key for fixing heroku

File: src/renderer/redux/modules/services/thunks.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      })
+      .catch(console.error);
+  };
+}
+
+export function getServiceChildren(taxId) {
+  return (dispatch, getState) => {
+    dispatch(
+      requestUrl(
+        `https://mofb-api.appspot.com/api/v1/taxonomy/${taxId}/children`,
```
Comment (mihiramin89) [inline src/renderer/redux/modules/services/thunks.js]: this is the same requestUrl as the above function. can be consolidated

File: src/renderer/redux/modules/services/thunks.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        GET,
+        {
+          successToast: 'successfully grabbed services',
+          errorToast: 'failed to fetch services',
+        }
+      )
+    )
+      .then(response => {
```
Comment (mihiramin89) [inline src/renderer/redux/modules/services/thunks.js]: might be able to shorten this to a single line

File: server.js [hunk 4]
```diff
@@ -10,6 +10,15 @@ const app = express();
 // the __dirname is the current directory from where the script is running
 app.use(express.static(distPath));
 
+app.use(function(req, res, next) {
```
Comment (mihiramin89) [inline server.js]: is this really needed? I think it can be removed.

Comment (vijayyadav06) [reply server.js]: yeah go ahead
